### ISSUE TITLE
About voice cloning

### ISSUE THREAD

**Issue by TheHonestBob** [TensorSpeech/TensorFlowTTS#659]
Thanks for your project, about voice cloning, the way of multi-speaker or loading pretrained models train on few other data can achieve voice cloning?

**Comment by ZDisket** [comment]
@TheHonestBob Yes. You can generally finetune Tacotron2 on 30 minutes of data, sometimes less if it's good quality and easy to learn (not much variance between samples, etc). FastSpeech2 is more tolerant of worse datasets, so it's worth looking into that. From my experience usually 6000 - 15k steps of Tac2 to get a good result and 15k - 30k of FS2. I'm preparing something that will include finetuning guide including a Tacotron2 model trained on phonemes, an efficiently finetuneable vocoder and better guided attention loss, so stay tuned for that.
If you have enough speakers that the total duration sums into enough for training from scratch (that is ,about 14 to 20 hours), you can do that too.

**Comment by Pydataman** [comment]
> @TheHonestBob Yes. You can generally finetune Tacotron2 on 30 minutes of data, sometimes less if it's good quality and easy to learn (not much variance between samples, etc). FastSpeech2 is more tolerant of worse datasets, so it's worth looking into that. From my experience usually 6000 - 15k steps of Tac2 to get a good result and 15k - 30k of FS2. I'm preparing something that will include finetuning guide including a Tacotron2 model trained on phonemes, an efficiently finetuneable vocoder and better guided attention loss, so stay tuned for that.
> If you have enough speakers that the total duration sums into enough for training from scratch (that is ,about 14 to 20 hours), you can do that too.

can achieve unseen speaker clone?
